### PR TITLE
exclude reward delivery from standard device validation

### PR DESCRIPTION
**Pull request by mochic**
Attempts to address #674

### CODE REVIEW

**Comment by saskiad** [comment]
I'd prefer to solve this by removing the `device_type` field from `RewardDelivery` (since it's not a device) and adding `name` that can have a default value of "Reward Delivery"

**Comment by mochic** [comment]
@saskiad does this look ok? `device_type` appears to be required by STIMULUS_DEVICES which uses it as a discriminator.

**Comment by dyf** [comment]
If you exclude delivery you won't be able to include reward delivery devices in the DAQ, e.g. the bottom of this:

https://github.com/AllenNeuralDynamics/aind-data-schema/pull/702

**Comment by saskiad** [comment]
I think we need to rework the validator so that stimulus devices isn't listed in standard devices and is handles separately.

File: src/aind_data_schema/models/devices.py [hunk 1]
```diff
@@ -774,6 +774,7 @@ class RewardSpout(Device):
 class RewardDelivery(AindModel):
     """Description of reward delivery system"""
 
+    name: str = "Reward delivery"
```
Comment (saskiad) [inline src/aind_data_schema/models/devices.py:777]: if we're keeping the device_type then don't add the name.

**Review by saskiad — CHANGES_REQUESTED** [review]
If we're keeping device_type for the Reward delivery then don't add the name

**Comment by saskiad** [comment]
> @saskiad If you exclude delivery from validation you won't be able to include reward delivery subdevices (spouts, solenoids) in the DAQ, e.g. the bottom of this:
> 
> #702

Agreed, removing it completely from validation is not an option. We can rework the validation to handle it better though.

File: src/aind_data_schema/models/devices.py (hunk 1 above)
Comment (saskiad) [reply src/aind_data_schema/models/devices.py:777]: Oh, nevermind. We need the name. I'd still prefer to change the validation than to add this, but that will take time.

**Pull request closed without merging**